drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -191,6 +191,9 @@ static gnrc_pktsnip_t *_make_netif_hdr(uint8_t *mhr)
         addr = gnrc_netif_hdr_get_dst_addr(hdr);
         for (int i = 0; i < dst_len; i++) {
             addr[i] = mhr[5 + (dst_len - i) - 1];
+            if(addr[i] != 0xff) {
+                hdr->flags |= GNRC_NETIF_HDR_FLAGS_BROADCAST;
```

Comment:
@daniel-k That's the wrong way around. Isn't it? In this case it is not a broadcast address... I thought about setting the flag where you set `broadcast = true;` before and `hdr->flags &= ~GNRC_NETIF_HRD_FLAGS_BROADCAST;` here.